### PR TITLE
Update mentoring.md

### PR DESCRIPTION
**Pull request by thomasantony12**
Changed some words and grammatical mistakes

### CODE REVIEW

File: tracks/python/exercises/perfect-numbers/mentoring.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ A simple optimization is to use `range(1, number // 2 + 1)` as the `for` loop `i
 This halves the complexity of the solution.
 
 For a more efficient solution, one can compute all the factors by using the smaller `range(1, int(math.sqrt(number)) + 1)` as the `iterable`.
-This solution is longer and more involved but significantly faster, reducing the complextiy from `O(n)` (_linear_) to `O(sqrt(n))` (_square root_).
+This solution is longer and more involved but significantly faster, reducing the complexities from `O(n)` (_linear_) to `O(sqrt(n))` (_square root_).
```
Comment (IsaacG) [inline tracks/python/exercises/perfect-numbers/mentoring.md:32]: "complexity" here should be singular, not plural.

File: tracks/python/exercises/perfect-numbers/mentoring.md [hunk 2]
```diff
@@ -48,6 +48,6 @@ def classify(number):
 ### Points to call out
 
 Students unfamiliar with `generator expressions` might write: `aliquot = sum([item for item in range(1, number) if number % item == 0])`.
-Note this first creates a `list` of factors in memory by iterating over the entire range, then iterates once more over the `list` to `sum()` its values.
-This is inefficent for both memory and processing time.
-Dropping the `[]` drops `list` creation and allows `sum()` to lazily process a `generator expression`, wich only requires a single iteration and a smaller memory footprint.
+Note that this first creates a `list` of factors in memory by iterating over the entire range, then iterating its values once more over the `list` to `sum()`.
```
Comment (IsaacG) [inline tracks/python/exercises/perfect-numbers/mentoring.md:51]: The original version of this line is correct as-is.

**Review by IsaacG — CHANGES_REQUESTED** [review]

File: tracks/python/exercises/perfect-numbers/mentoring.md (hunk 1 above)
Comment (thomasantony12) [reply tracks/python/exercises/perfect-numbers/mentoring.md:32]: ok I will make the changes

File: tracks/python/exercises/perfect-numbers/mentoring.md (hunk 2 above)
Comment (thomasantony12) [reply tracks/python/exercises/perfect-numbers/mentoring.md]: Changed to original version.

**Pull request closed without merging**